File: object_oriented_programming/uber_practice/python/uber_pool.py
#!/usr/bin/python3
""" UberPool is a subclass of Car class
    """

from car import Car


class UberPool (Car):
    brand = str
    model = str

    # UberPool constructor should hava a least the same parameters
    # of its superclass Car, then use 'super' to call them.
    def __init__(self, license, driver, brand, model):
        super().__init__(license, driver)
        self.brand = brand
        self.model = model

File: object_oriented_programming/uber_practice/java/UberPool.java
/**
 * UberPool is a subclass of Car class
 */

class UberPool extends Car {

    String brand;
    String model;

    // UberPool constructor should containg at least the same parameters of its
    // superclass, then use 'super' to call the superclass constructor
    public UberPool(String license, Account driver, String brand, String model) {
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }
}
